### DISCUSSION TITLE
Add-on Store: More friendly organization of the Actions/context menu

### DISCUSSION THREAD

**Discussion by XLTechie** [nvaccess/nvda#14999] [Ideas]
### Is your feature request related to a problem? Please describe.

There may be a better organization for the Actions/context menu used for add-ons in the Add-on Store.

The current order is:

* Disable / Enable
* Remove / Install
* Help
* Homepage
* License
* Source Code

### Describe the solution you'd like

I think that offering help options before action items, is probably a better UX.

* Help
* Homepage
* Disable / Enable
* Remove / Install
* License
* Source Code

### Describe alternatives you've considered

Could be left as is.

### Additional context

This is certainly not high priority, but I think users are more likely to be using that menu (for installed add-ons) to get help, before any of the other choices.

**Comment by seanbudd** [comment]
We think installing is the primary action people want to perform when it's available.
We also think actions for add-ons are a priority over pages/links.
Instead the following order is suggested:

- Install
- Disable / Enable
- Remove
- Help
- Homepage
- License
- Source Code

**Comment by Qchristensen** [comment]
I did have feedback from a user that they would really like shortcut keys directly to actions (eg install) rather than needing to go via the actions button.

I can see where if I know what I want, I would go through my list of add-ons, find the one I want to remove, and want to press say alt+r to remove it.  Or go through the store, find an add-on I already know about, and want to install it with a shortcut key.
